pkg/notifications/writer.go

```diff
@@ -0,0 +1,10 @@
+package notifications
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// Writer is capable of writing notifications. The notification may be written to an underlying storage.
+type Writer interface {
```

Comment:
I guess this is no longer needed with https://github.com/stackrox/stackrox/pull/7509?